8-typography.rst

```diff
@@ -760,7 +760,7 @@ Numbers, measurements, and math
 
 			<p>The 1st, 2nd, 3rd, 4th.</p>
 
-#.	Numbers in a non-mathematical context are spelled out if they are less than or equal to 100. Numbers over 100 are set with digits.
```

Comment:
Let's keep this though, for clarity

Reply:
Good grief, sorry about that. I deleted my added sentence and apparently also picked up the previous one by mistake. Fixed.